airflow-core/src/airflow/ui/openapi-gen/requests/types.gen.ts

```diff
@@ -1102,19 +1107,26 @@ export type JobResponse = {
 };
 
 /**
- * Menu Item for responses.
+ * Define all menu items defined in the menu.
  */
-export type MenuItem = {
-  text: string;
-  href: string;
-};
+export type MenuItem =
+  | "Assets"
+  | "Asset Events"
```

Comment:
Why Asset Events and not Dag Runs or Task Instances?

We don't have a single asset events page yet.

Reply:
Or did you mean just `Events` for what used to be called Audit Log?

Reply:
Yes I meant `Events`, I dont know why I named it `Asset Events` (has it been renamed recently?). Let me update it. I do not focus on pages, I focus on links we have on the menu, `Dag Runs` or `Task Instance` are not part of the menu, that is why they are not part of this list.